utils/test/deprecation.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Internal dependencies
+ */
+import { deprecated } from '../deprecation';
+
+describe( 'deprecated', () => {
```

Comment:
Love those tests, they are so fun to read and perfectly explain all the logic hidden in the tested method ❤️